TOOLS/lua/autoload.lua

```diff
@@ -89,17 +91,17 @@ EXTENSIONS_IMAGES = Set {
 }
 
 function split_option_exts(video, audio, image)
-    if video then o.additional_video_exts = Split(o.additional_video_exts) end
-    if audio then o.additional_audio_exts = Split(o.additional_audio_exts) end
-    if image then o.additional_image_exts = Split(o.additional_image_exts) end
+    if video then EXTENSIONS_VIDEO = SetUnion(EXTENSIONS_VIDEO, Split(o.additional_video_exts)) end
+    if audio then EXTENSIONS_AUDIO = SetUnion(EXTENSIONS_AUDIO, Split(o.additional_audio_exts)) end
+    if image then EXTENSIONS_IMAGES = SetUnion(EXTENSIONS_IMAGES, Split(o.additional_image_exts)) end
```

Comment:
Adding the extensions from the options to the `EXTENSIONS_*` sets breaks runtime updates, because entries can be added this way, but not removed.

Reply:
Indeed, I only tested that the runtime add would work, ignoring the deleted scenes.

Reply:
So we need to address the support for removing scenes here